treescript/src/treescript/mercurial.py

```diff
@@ -222,6 +222,11 @@ def build_hg_environment():
     return tags
 
 
+async def get_revision(config, repo_path, branch):
```

Comment:
`s/branch/revset/` here, I think. This can be any [revset](https://www.mercurial-scm.org/repo/hg/help/revsets).

Reply:
If we don't know the revision, how would we pass it in? I think the use-case here is when we're given a bookmark